.github/workflows/ci.yml

```diff
@@ -10,22 +10,29 @@ on:
 
 jobs:
   lint-and-types:
+    if: github.event.pull_request.draft == false
+
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [23.x]
+        node:
+          - version: 23.x
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # 4.2.2
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-    - run: npm ci
-    - run: npm run lint
-    - run: npm run format
-    - run: npm run test:types
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # 4.2.2
+      - name: Set up Node.js ${{ matrix.node.version }}
+        uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # 4.1.0
+        with:
+          node-version: ${{ matrix.node.version }}
+          cache: 'npm'
+      - run: npm ci
+      - run: node --run lint
+      - run: node --run format
```

Comment:
maybe use prebuilt GA by biome because it's include a reporter so on PR we see where there are something wrong.

https://biomejs.dev/recipes/continuous-integration/#github-actions

Reply:
🤔 That would make CI and local behave differently. I think we should avoid that.

Reply:
it's run command "biome ci ./".

Eh wait, this ins't on `scripts` other things can be added